### ISSUE TITLE
elpy-multiedit-python-symbol-at-point continues to edit symbol resulting in unexpected edits

### ISSUE THREAD

**Issue by cammil** [jorgenschaefer/elpy#1108]
After calling `elpy-multiedit-python-symbol-at-point` it appears that, the usages of that symbol may continue to edit long after I would *expect* them not to.

For example, after editing, and then navigating elsewhere (off the symbol) does not cancel the edit. It seems that `C-g` also does not cancel the edit. The result is that any later editing of the text of the symbol continues to affect other uses. This is very confusing and has caused unexpected changes to my code.

Perhaps I am not cancelling the active multiedit, but if so, it is not clear from the documentation how one should do that. Perhaps there is something else I should know, but that too is not evident from the documentation.

Not sure if this is a bug, documentation issue, or my misunderstanding!

**Comment by jorgenschaefer** [comment]
> I do not know how to cancel the multiedit except for perhaps calling it again.

Yes, multiedit remains active (and should highlight all affected locations) until you call the command again to disable these regions. Does that solve the confusion?

**Comment by andravin** [comment]
> call the command again to disable these regions.

That is undocumented behavior.
